code/game/objects/structures/industrial_lift.dm

```diff
@@ -404,6 +404,7 @@ GLOBAL_LIST_EMPTY(lifts)
 	//kind of a centerpiece of the station, so pretty tough to destroy
 	armor = list(MELEE = 80, BULLET = 80, LASER = 80, ENERGY = 80, BOMB = 100, BIO = 80, RAD = 80, FIRE = 100, ACID = 100)
 	resistance_flags = FIRE_PROOF | ACID_PROOF
+	var/obj/machinery/computer/tram_controls/console //console sets this in late initialize
```

Comment:
these dont seem to register for the console being deleted, which would cause a hard del

Reply:
What @Kylerace said + add DMDoc